src/cljs/org/broadinstitute/firecloud_ui/page/billing/billing_management.cljs

```diff
@@ -25,11 +67,18 @@
        [table/Table
         {:columns [{:header "Project Name" :starting-width 400 :sort-by #(% "projectName")
                     :content-renderer
-                    (fn [{:strs [projectName role]}]
-                      (if (= role "Owner")
-                        (style/create-link {:text projectName
-                                            :onClick #((:on-select props) projectName)})
-                        projectName))}
+                    (fn [{:strs [projectName role status]}]
```

Comment:
the last commit on the rawls side changed "status" to "creationStatus" - so the keys in the UI need a corresponding change. FYI, Rawls swagger also needs an update for status->creationStatus.